docs/get_started.rst

```diff
@@ -1,17 +1,369 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Verify project initialization was successful by verifying that ``cumulusci.yml`` exists and has contents.
+
+.. code-block:: console
+
+    $ cat cumulusci.yml
+    project:
+        name: SampleProjectName
+        package:
+            name: SamplePackageName
+            namespace: sampleNamespace
+        .
+        .
+        .
+
+The ``cumulusci.yml`` file configures your project-specific tasks, flows, and customizations.
+
+For more information regarding configuration, check out our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can even add and commit it to your Git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+With your ``cumulusci.yml`` file committed, create a repository on GitHub for your new project and push your changes there.
+There are multiple ways to do this:
+
+* Our `Community Project Development with CumulusCI <https://trailhead.salesforce.com/content/learn/modules/community-project-development-with-cumulusci?trail_id=build-applications-with-cumulusci>`_ module covers GitHub Desktop in the "Set Up the Project" section.
+* If you prefer the command line, GitHub has excellent guides on both `git remote <https://github.com/git-guides/git-remote>`_ and `git push <https://github.com/git-guides/git-push>`_.
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+Converting an existing Salesforce project to use CumulusCI may follow a number of different paths, depending on whether you're practicing the Org Development Model or the Package Development Model; whether or not you're already developing in scratch orgs; and the complexity of your project's dependencies on the org environment.
+
+If you're developing in scratch orgs, then you likely only need to do `project setup`_ and `org shape setup`_.
+
+If you're developing in persistent orgs, then you likely want to go through **all** of the following sections. Your experience may vary. 
+
+You're welcome to discuss project conversion in the `CumulusCI (CCI) Trailblazer group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9Z>`_.
+
+
+Project Setup
+^^^^^^^^^^^^^
+Prior to capturing your project's Metadata you need to set up the project's root directory. These steps include:
+
+* Creating a project directory and initializing it as a Git repository as outlined in `starting a new CumulusCI project`_.
+* Initializing the directory as a CumulusCI project as outlined in `project initialization`_.
+
+
+
+Metadata Capture
+^^^^^^^^^^^^^^^^
+The following assumes that your project currently lives in a persistent org, such as a Developer Edition org or a sandbox. We recommend a retrieve of MetaData via the Metadata API (via ``sfdx``), followed by converting the source format from "metadata" to "``sfdx``".
```

Comment:
Retrieve as noun. (Makes sense, but just want to confirm that we want to phrase it that way, as well as whether or not it should be in command font.) MetaData oddly capitalized. Confirm that we want quotes around metadata and sfdx.

Reply:
```suggestion
Metadata Capture
^^^^^^^^^^^^^^^^
The following assumes that your project currently lives in a persistent org, such as a Developer Edition org or a sandbox. We recommend retrieving metadata via the Metadata API (using the Salesforce CLI), followed by converting the source from Metadata API to SFDX format.
```

Reply:
Also I know this wording didn't come from you, Justin, but we've been trying to be consistent about talking about "retrieving" metadata rather than "capturing" it, so should avoid Capture in the title of this section. Capture makes it sound like it's on the lam.

Reply:
@davisagli: So should the header be "Retrieving Metadata"?